examples/travis.example.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - 4
+  - 5
+
+addons:
+  firefox: 'latest'
```

Comment:
They run in Chrome on circle. Travis does not provide Chrome and I don't know how to set things up so that Chrome runs on Travis